src/psfmachine/machine.py

```diff
@@ -682,7 +693,7 @@ def build_time_model(
             mpc1_smooth, mpc2_smooth = bspline_smooth(
                 [mpc1, mpc2],
                 x=self.time,
-                do_segments=segments,
+                do_segments=True,
```

Comment:
Always?

Reply:
![image](https://user-images.githubusercontent.com/10449555/176941651-e8b0a71c-4446-4203-ab8b-1e61db64bbe5.png)

Reply:
Can we add a log.warning that explains what is happening here